scripts/logs/logs-json.sh

```
@@ -0,0 +1,273 @@
+#!/bin/bash
+# JSON log pretty-printer and analyzer
+# Usage: ./logs-json.sh [--field FIELD] [--config] [--help]
+
+# Colors
+GREEN='\033[0;32m'
+BLUE='\033[0;34m'
+CYAN='\033[0;36m'
+YELLOW='\033[1;33m'
+WHITE='\033[1;37m'
```

Shellcheck (0.10.0)

[warning] WHITE appears unused. Verify use (or export if used externally).

(SC2034)